aries-site/src/pages/templates/page-layouts.mdx

```diff
@@ -145,6 +146,22 @@ Page Content will be left aligned to the edge of the screen.
 - Maps 
 - Tables
 
+### Page Container Widths Demonstration
+
+Open the following example in "fullscreen mode" and resize your browser window to see how 
+each [page container width's](/templates/page-layouts#page-container-widths) maximum width, 
```

Comment:
I liked during our demo this morning how you could see the break points I know they were for your testing purposes but wondering if we can incorporate may help users

Reply:
we can think on this just something that came to mind

Reply:
I like having it too. Let's play around with it.